studio/components/layouts/ProjectLayout/LayoutHeader/ProjectDropdown.tsx

```diff
@@ -15,9 +15,16 @@ import { IS_PLATFORM, PROJECT_STATUS } from 'lib/constants'
 // highest common route with just projectRef in the router queries.
 
 const sanitizeRoute = (route: string, routerQueries: ParsedUrlQuery) => {
-  let queryArray = Object.entries(routerQueries)
+  const queryArray = Object.entries(routerQueries)
+
   if (queryArray.length > 1) {
-    return route.split('/').slice(0, 4).join('/')
+    // account for query string, if exists (example: /logs/explorer?q=select...)
+    const hasQueryString = queryArray.filter(([key]) => key === 'q').length > 0
```

Comment:
`some()` works good for this:
```suggestion
    const hasQueryString = queryArray.some(([key]) => key === 'q')
```